src/containerapp/azext_containerapp/tests/latest/utils.py

```diff
@@ -327,6 +331,8 @@ def create_and_verify_containerapp_create_and_update(
             create_cmd += f" --source \"{source_path}\""
         if artifact_path:
             create_cmd += f" --artifact \"{artifact_path}\""
+        if build_env_vars:
```

Comment:
I think it should check the result of the `build_env_vars`, I cannnot find the checking.

Reply:
build_env_vars is saved in Build resource, currently there's no CLI for build resource. I will add build_env_vars verification after CLI  `containerapp build show` is ready.